cwms-data-api/src/test/resources/cwms/cda/data/sql/load_testusers.sql

```diff
@@ -1,8 +1,9 @@
-declare
-    group_list cwms_20.char_32_array_type;
 begin
-    group_list := cwms_20.CHAR_32_ARRAY_TYPE('CWMS Users');
-    cwms_20.cwms_sec.create_user('user','blah',group_list, 'HQ');    
-    cwms_20.cwms_sec.create_user('user2','blah',NULL,'SPK');    
-    cwms_20.cwms_sec.create_user('s0webtest','&password.',group_list,'HQ');    
+    create user "l2hectest" identified by "l2hectest";
```

Comment:
None of these users should be actual oracle users in these tests.

Reply:
Except maybe s0webtest, the rest operate as "cac only".

Reply:
thanks for this catch, I didn't mean to change this file.